person/views.py

```diff
@@ -118,7 +121,32 @@ def asem_user_details(request, asem_user_id):
     asem_user.status = choices_dict['status'][asem_user.status]
     asem_user.own_home = choices_dict['housing_type'][asem_user.own_home]
 
-    return render(request, 'asem_user/asem_user_details.html', {'asem_user': asem_user})
+    fields = [f for f in ASEMUser._meta.get_fields() if f.name not in ["id", "photo", "password", "user_type", "name", "surname", "service", "ong", "person_ptr"]]
+    info = [getattr(asem_user, f.name) for f in fields]
+
+    fields_info = dict(zip([f.verbose_name for f in fields], info))
+    
+    items = list(fields_info.items())
+
+    for item in items:
+        if(item[1] == True):
+            items[items.index(item)] = (item[0], "Sí")
+        elif(item[1] == False):
+            items[items.index(item)] = (item[0], "No")
+        elif(item[0] == "Género" and item[1] != None):
+            choices = Volunteer._meta.get_field('sex').choices
+            value = [choice[1] for choice in choices if choice[0] == item[1]][0]
+            items[items.index(item)] = (item[0], value)
+        elif(item[0] == "Tiempo de dedicación"):
+            items[items.index(item)] = (item[0], str(item[1]) + " horas")
+
+    items = [item for item in items if item[1] != None and item[1] != '' and item[1] != []]
+
+    mid = math.ceil(len(items) / 2)
+    
+    context = {'asem_user': asem_user, 'info_left': items[:mid], 'info_right': items[mid:]}
+
+    return render(request, 'users/details.html', context)
```

Comment:
In Asem User when you put "Tamaño unidad familiar" to 1 it sais "Sí" instead. (In red🟥)

Also, as a suggestion the question fields can be changed to "Vehículo propio" for example. But as it is ok too, so if you have time you can improve it. (In yellow🟨)

![image](https://user-images.githubusercontent.com/53529939/230734356-4db12495-a6b7-4525-b428-53473c74dbae.png)

Reply:
I will try to fix the red thing. I have the feeling that this is involved with the backend side, because the only way that I show there "Sí" is when the value of the database is True. I will check this anyway.

As for the yellow thing, I could fix it, but I already overpassed in hours this issue, so I won't change it.